jans-cedarling/bindings/cedarling_python/cedarling_python.pyi

```diff
@@ -11,152 +11,106 @@ class BootstrapConfig:
-            - "at_jti_validation" (str): Access token JWT ID validation status. Defaults to "enabled".
-            - "at_nbf_validation" (str): Access token "not before" validation status. Defaults to "enabled".
-            - "at_exp_validation" (str): Access token expiration validation status. Defaults to "enabled".
-            - "idt_iss_validation" (str): ID token issuer validation status. Defaults to "enabled".
-            - "idt_sub_validation" (str): ID token subject validation status. Defaults to "enabled".
-            - "idt_exp_validation" (str): ID token expiration validation status. Defaults to "enabled".
-            - "idt_iat_validation" (str): ID token issued-at validation status. Defaults to "enabled".
-            - "idt_aud_validation" (str): ID token audience validation status. Defaults to "enabled".
-            - "userinfo_iss_validation" (str): User info issuer validation status. Defaults to "enabled".
-            - "userinfo_sub_validation" (str): User info subject validation status. Defaults to "enabled".
-            - "userinfo_aud_validation" (str): User info audience validation status. Defaults to "enabled".
-            - "userinfo_exp_validation" (str): User info expiration validation status. Defaults to "enabled".
-            - "id_token_trust_mode" (str): Trust mode for ID tokens. Defaults to "strict".
-            - "lock" (str): Lock mechanism status. Defaults to "disabled".
-            - "lock_master_configuration_uri" (str or None): Master configuration URI for locks. Defaults to None.
-            - "dynamic_configuration" (str): Dynamic configuration status. Defaults to "disabled".
-            - "lock_ssa_jwt" (str or None): JWT for SSA locks. Defaults to None.
-            - "audit_log_interval" (int): Interval for audit logs in seconds. Defaults to 0.
-            - "audit_health_interval" (int): Interval for health audit in seconds. Defaults to 0.
-            - "audit_health_telemetry_interval" (int): Interval for telemetry health audit in seconds. Defaults to 0.
-            - "listen_sse" (str): Server-Sent Events listening status. Defaults to "disabled".
-
-    Returns:
-        An instance of the class.
-
-    Raises:
-        KeyError: If any required configuration key is missing.
-        ValueError: If a provided value is invalid or extraction fails.
-    """
-
-    def disable_all_jwt_validation(self) -> None: ...
-
+    ) -> None:
+        """
+        Initialize a new instance of the class with configuration options.
+
+        Args:
+            options (dict): A dictionary containing configuration key-value pairs.
+
+                Required keys:
+                    - "application_name" (str): The name of the application.
+                    - "policy_store_id" (str): The ID of the policy store.
+
+                Optional keys:
+                    - "policy_store_uri" (str, optional): URI of the policy store. Defaults to None.
```

Comment:
I thought that we don't use keys in snake_case already

Reply:
fixed here [49b5e36](https://github.com/JanssenProject/jans/pull/10350/commits/49b5e362dab383d4b5bd1e5821aa15cab392626a)